camera:
  frame_id: base_link
cmd:
  use_comp_id_system_control: false
conn:
  heartbeat_rate: 1.0
  system_time_rate: 1.0
  timeout: 10.0
  timesync_rate: 10.0
distance_sensor:
  hrlv_ez4_pub:
    field_of_view: 0.0
    frame_id: hrlv_ez4_sonar
    id: 0
    orientation: PITCH_270
    send_tf: true
    sensor_position:
      x: 0.0
      y: 0.0
      z: -0.1
  laser_1_sub:
    id: 3
    orientation: PITCH_270
    subscriber: true
  lidarlite_pub:
    field_of_view: 0.0
    frame_id: lidarlite_laser
    id: 1
    orientation: PITCH_270
    send_tf: true
    sensor_position:
      x: 0.0
      y: 0.0
      z: -0.1
  sonar_1_sub:
    horizontal_fov_ratio: 1.0
    id: 2
    orientation: PITCH_270
    subscriber: true
    vertical_fov_ratio: 1.0
fake_gps:
  eph: 2.0
  epv: 2.0
  fix_type: 3
  geo_origin:
    alt: 408.0
    lat: 47.3667
    lon: 8.55
  gps_rate: 5.0
  mocap_transform: true
  satellites_visible: 5
  tf:
    child_frame_id: fix
    frame_id: odom
    listen: false
    rate_limit: 10.0
    send: true
  use_mocap: true
  use_vision: false
global_position:
  child_frame_id: base_link
  frame_id: odom
  gps_uere: 1.0
  rot_covariance: 99999.0
  tf:
    child_frame_id: base_link
    frame_id: odom
    global_frame_id: earth
    send: false
  use_relative_alt: true
image:
  frame_id: px4flow
imu:
  angular_velocity_stdev: 0.0003490659  # 0.02 degrees
  frame_id: base_link
  linear_acceleration_stdev: 0.0003
  magnetic_stdev: 0.0
  orientation_stdev: 1.0
landing_target:
  camera:
    fov_x: 2.0071286398
    fov_y: 2.0071286398
  image:
    height: 480
    width: 640
  land_target_type: VISION_FIDUCIAL
  listen_lt: false
  mav_frame: LOCAL_NED
  target_size:
    x: 0.3
    y: 0.3
  tf:
    child_frame_id: camera_center
    frame_id: landing_target
    listen: false
    rate_limit: 10.0
    send: true
local_position:
  frame_id: odom
  tf:
    child_frame_id: base_link
    frame_id: odom
    send: true
    send_fcu: false
mission:
  pull_after_gcs: true
  use_mission_item_int: true
mocap:
  use_pose: true
  use_tf: false
mount:
  debounce_s: 4.0
  err_threshold_deg: 10.0
  negate_measured_pitch: false
  negate_measured_roll: false
  negate_measured_yaw: false
odometry:
  fcu:
    map_id_des: map
    odom_child_id_des: base_link
    odom_parent_id_des: odom
px4flow:
  frame_id: px4flow
  ranger_fov: 0.118682
  ranger_max_range: 5.0
  ranger_min_range: 0.3
safety_area:
  p1:
    x: 1.0
    y: 1.0
    z: 1.0
  p2:
    x: -1.0
    y: -1.0
    z: -1.0
setpoint_accel:
  send_force: false
setpoint_attitude:
  reverse_thrust: false
  tf:
    child_frame_id: target_attitude
    frame_id: odom
    listen: false
    rate_limit: 50.0
  use_quaternion: false
setpoint_position:
  mav_frame: LOCAL_NED
  tf:
    child_frame_id: target_position
    frame_id: odom
    listen: false
    rate_limit: 50.0
setpoint_raw:
  thrust_scaling: 1.0
setpoint_velocity:
  mav_frame: LOCAL_NED
startup_px4_usb_quirk: false
sys:
  disable_diag: false
  min_voltage: 10.0
tdr_radio:
  low_rssi: 40
time:
  time_ref_source: fcu
  timesync_avg_alpha: 0.6
  timesync_mode: MAVLINK
vibration:
  frame_id: base_link
vision_pose:
  tf:
    child_frame_id: vision_estimate
    frame_id: odom
    listen: false
    rate_limit: 10.0
vision_speed:
  listen_twist: true
  twist_cov: true
wheel_odometry:
  child_frame_id: base_link
  count: 2
  frame_id: odom
  send_raw: true
  send_twist: false
  tf:
    child_frame_id: base_link
    frame_id: odom
    send: true
  use_rpm: false
  vel_error: 0.1
  wheel0:
    radius: 0.05
    x: 0.0
    y: -0.15
  wheel1:
    radius: 0.05
    x: 0.0
    y: 0.15
